python/PyQt/PyQt/QtCore.py.in

```diff
@@ -72,5 +72,64 @@ QVariant.__hash__ = __hash__
 
 if (QT_VERSION >= 0x060000):
     NULL = None
+    # monkey patch in QVariant Types
```

Comment:
Why not relying on the migration script pyqt5_to_pyqt6 to change this one directly in the python script ?

Reply:
Oh yes, I forgot